### PR TITLE
update readme of oceanbase-ce

### PR DESCRIPTION
**Pull request by chris-sun-star**
## Summary

update readme of oceanbase-ce

### CODE REVIEW

File: oceanbase-ce/README.md [hunk 1]
```diff
@@ -50,78 +43,65 @@ boot success!
 
 ## Connect to an OceanBase instance
 
-The `oceanbase-ce` image contains `obclient` (OceanBase Database Client) and the default connection script `ob-mysql`.
+The `oceanbase-ce` image contains `obclient` (OceanBase Database Client) and the default connection script `ob-mysql`. You may refer to the following commands to connect to OceanBase cluster.
```
Comment (whhe) [inline oceanbase-ce/README.md]: I prefer to remove the ob-mysql script and use normal connection params of mysql client or obclient. The ob-mysql here introduce more complexity.  We can show `docker exec -it oceanbase-ce obclient -uroot ...` here.

File: oceanbase-ce/README.md [hunk 2]
```diff
@@ -50,78 +43,65 @@ boot success!
...
-| OB_SYS_PASSWORD        |                      | The oceanbase root user password of sys tenant                                                                                                                                                                                                                                                                                                                                                                                                            |
+| OB_SYS_PASSWORD         |                      | The oceanbase root user password of sys tenant                                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (whhe) [inline oceanbase-ce/README.md:77]: OB_TENANT_PASSWORD is missing.

File: oceanbase-ce/README.md [hunk 3]
```diff
@@ -1,47 +1,40 @@
...
+# deploy an instance to use the full resource of container
 docker run -p 2881:2881 --name oceanbase-ce -e MODE=normal -d oceanbase/oceanbase-ce
 
-# deploy a quick-start instance in any mode as desired to the current container
+# deploy an instance using fastboot mode
 docker run -p 2881:2881 --name oceanbase-ce -e MODE=slim -d oceanbase/oceanbase-ce
 
-# deploy an instance and execute init sqls
+# deploy an instance and execute init sqls after bootstrap
 docker run -p 2881:2881 --name oceanbase-ce -v {init_sql_folder_path}:/root/boot/init.d -d oceanbase/oceanbase-ce
```
Comment (whhe) [inline oceanbase-ce/README.md:33]: Add a note to remind users not to change passwords by init sql, but to use OB_TENANT_PASSWORD instead.

File: oceanbase-ce/README.md (hunk 1 above)
Comment (chris-sun-star) [reply oceanbase-ce/README.md]: Yes, so do I

File: oceanbase-ce/README.md (hunk 3 above)
Comment (chris-sun-star) [reply oceanbase-ce/README.md:33]: OK

**Review by whhe — APPROVED** [review]

**Pull request merged**